### PR TITLE
Add batch processing option and pre-commit hooks

### PR DESCRIPTION
**Pull request by brmdv**
This feature allows you to format (or check) more than one file. I understand that this is not a common use case, as most projects typically have a single changelog file. However, I frequently work with repositories that contain folders, each with their own CHANGELOG.md file.

I first tried to update the existing `format` subcommand, but I could not get it working properly, as allowing multiple args for `--path` interferes with it being the default action.

The need for this came up when I tried to create a pre-commit-hooks.yml file, and I discovered that it does not have an option to pass files one by one to the command – it will always pass all changed files that match the pattern to the entry.

### CODE REVIEW

**Review by mdwint — COMMENTED** [review]
I don't see the code that implements the batch command. Did you forget to commit it?

Aside from that, I'm not a fan of creating a new command just for this. There must be a way of passing multiple values to `--path`, or passing the `--path` argument multiple times using `action="append"`.

**Comment by brmdv** [comment]
Seems that I accidentally removed it from the commit while doing some clean-up, indeed. I put it back for now. 

Passing multiple values to `--path` interfered with having `format` as the default action when I tried it. I will have a go with `action='append'`, when I come back to this.

**Comment by brmdv** [comment]
I tried again to allow multiple arguments for `--path`, and it turns out that I came to conclusions too early. The problem was not `format` being the default action, but just that I forgot to make the default path a single-item list.

File: ocdc/__main__.py [hunk 1]
```diff
@@ -29,7 +31,7 @@ def build_arg_parser() -> argparse.ArgumentParser:
         help=(
             "Don't modify the file, only report the status.\n"
             "Exit code 0 means nothing would change.\n"
-            "Exit code 1 means the file would be reformatted."
+            "Exit code 1 means the at least one of the files would be reformatted."
```
Comment (mdwint) [inline ocdc/__main__.py]: `the at` -> `at`

**Comment by mdwint** [comment]
Thanks for adding this!

**Pull request merged**